README.md

```diff
@@ -139,13 +162,39 @@ Here is an example:
         "use_cuda": true,
         "batch_size": 8,
         "skip_elmo": false,
-        "knn": 500,
         "publications_path": "./",
         "submissions_path": "./"
     }
 }
 ```
 
+#### ELMo specific parameters (duplicate detection):
+- `model_params.other_submissions_path`: When running ELMo, this is where the embedded abstracts/titles of the other Submissions are stored.
+All the other parameters are the same as in the affinity scores.
+- `model_params.skip_same_id`: Boolean value. If `true` Submissions with the same ID are skipped. This parameter is useful when trying to identify duplicates within the same Submissions. In those cases, if the ID is the same, the Subission is the same but it's not a duplicate.
+
+Here is an example:
+```
+{
+    "name": "duplicate_detection",
+    "dataset": {
+        "directory": "./"
+    },
+    "model_params": {
+        "scores_path": "./",
+        "use_title": false,
+        "use_abstract": true,
+        "use_cuda": true,
+        "batch_size": 4,
+        "knn": 10,
+        "skip_same_id": false,
+        "skip_elmo": false,
+        "submissions_path": "./",
+        "other_submissions_path": "./"
```

Comment:
Oh and one more thing I realized. I had to copy over the data in the dir `experiments_dir/submissions` into a dir `experiments_dir/other_submissions` that I had to create. It would be great if I did not have to duplicate the data to be able to do duplicate detection within the conference. Maybe other_submissions can be an optional config param and the absence of which would indicate an intra-conference duplicate detection. We would also not have to run embedding on the same submissions twice thereby saving some execution time as well. What are your thoughts

Reply:
Yup, that's a good idea. I was being lazy haha. I'll add that option. I think that would address your comment above?